base/ltfssaxes.dtx

```diff
@@ -192,503 +248,3154 @@
+\DeclareFontSeriesChangeRule {ubc}{el}  {elc}  {}
+\DeclareFontSeriesChangeRule {ubc}{l}   {lc}   {}
+\DeclareFontSeriesChangeRule {ubc}{sl}  {slc}  {}
+\DeclareFontSeriesChangeRule {ubc}{sb}  {sbc}  {bc}
+\DeclareFontSeriesChangeRule {ubc}{b}   {bc}   {}
+\DeclareFontSeriesChangeRule {ubc}{eb}  {ebc}  {bc}
+\DeclareFontSeriesChangeRule {ubc}{ub}  {ubc}  {bc}
+\DeclareFontSeriesChangeRule {ubc}{uc}  {ubuc} {}
+\DeclareFontSeriesChangeRule {ubc}{ec}  {ubec} {}
+\DeclareFontSeriesChangeRule {ubc}{c}   {ubc}  {}
+\DeclareFontSeriesChangeRule {ubc}{sc}  {ubsc} {}
+\DeclareFontSeriesChangeRule {ubc}{sx}  {ubsx} {}
+\DeclareFontSeriesChangeRule {ubc}{x}   {ubx}  {}
+\DeclareFontSeriesChangeRule {ubc}{ex}  {ubex} {}
+\DeclareFontSeriesChangeRule {ubc}{ux}  {ubux} {}
+%    \end{macrocode}
+%
+%    \begin{macrocode}
+\DeclareFontSeriesChangeRule {ubsc}{ul} {ulsc} {}
+\DeclareFontSeriesChangeRule {ubsc}{el} {elsc} {}
+\DeclareFontSeriesChangeRule {ubsc}{l}  {lsc}  {}
+\DeclareFontSeriesChangeRule {ubsc}{sl} {slsc} {}
+\DeclareFontSeriesChangeRule {ubsc}{sb} {sbsc} {bsc}
+\DeclareFontSeriesChangeRule {ubsc}{b}  {bsc}  {}
+\DeclareFontSeriesChangeRule {ubsc}{eb} {ebsc} {bsc}
+\DeclareFontSeriesChangeRule {ubsc}{ub} {ubsc} {bsc}
+\DeclareFontSeriesChangeRule {ubsc}{uc} {ubuc} {}
+\DeclareFontSeriesChangeRule {ubsc}{ec} {ubec} {}
+\DeclareFontSeriesChangeRule {ubsc}{c}  {ubc}  {}
+\DeclareFontSeriesChangeRule {ubsc}{sc} {ubsc} {}
+\DeclareFontSeriesChangeRule {ubsc}{sx} {ubsx} {}
+\DeclareFontSeriesChangeRule {ubsc}{x}  {ubx}  {}
+\DeclareFontSeriesChangeRule {ubsc}{ex} {ubex} {}
+\DeclareFontSeriesChangeRule {ubsc}{ux} {ubux} {}
+%    \end{macrocode}
+%
+%    \begin{macrocode}
+\DeclareFontSeriesChangeRule {ub}{uc}   {ubuc} {}
+\DeclareFontSeriesChangeRule {ub}{ec}   {ubec} {}
+\DeclareFontSeriesChangeRule {ub}{c}    {ubc}  {}
+\DeclareFontSeriesChangeRule {ub}{sc}   {ubsc} {}
+\DeclareFontSeriesChangeRule {ub}{sx}   {ubsx} {}
+\DeclareFontSeriesChangeRule {ub}{x}    {ubx}  {}
+\DeclareFontSeriesChangeRule {ub}{ex}   {ubex} {}
+\DeclareFontSeriesChangeRule {ub}{ux}   {ubux} {}
+%    \end{macrocode}
+%
+%    \begin{macrocode}
+\DeclareFontSeriesChangeRule {ubsx}{ul} {ulsx} {}
+\DeclareFontSeriesChangeRule {ubsx}{el} {elsx} {}
+\DeclareFontSeriesChangeRule {ubsx}{l}  {lsx}  {}
+\DeclareFontSeriesChangeRule {ubsx}{sl} {slsx} {}
+\DeclareFontSeriesChangeRule {ubsx}{sb} {sbsx} {bsx}
+\DeclareFontSeriesChangeRule {ubsx}{b}  {bsx}  {}
+\DeclareFontSeriesChangeRule {ubsx}{eb} {ebsx} {bsx}
+\DeclareFontSeriesChangeRule {ubsx}{ub} {ubsx} {bsx}
+\DeclareFontSeriesChangeRule {ubsx}{uc} {ubuc} {}
+\DeclareFontSeriesChangeRule {ubsx}{ec} {ubec} {}
+\DeclareFontSeriesChangeRule {ubsx}{c}  {ubc}  {}
+\DeclareFontSeriesChangeRule {ubsx}{sc} {ubsc} {}
+\DeclareFontSeriesChangeRule {ubsx}{sx} {ubsx} {}
+\DeclareFontSeriesChangeRule {ubsx}{x}  {ubx}  {}
+\DeclareFontSeriesChangeRule {ubsx}{ex} {ubex} {}
+\DeclareFontSeriesChangeRule {ubsx}{ux} {ubux} {}
```

Comment:
there is some argument here to set the 4th argument to the first  in such cases where to the existing series  contains bold since we know this exists and the requested combination doesn't